tensorflow/python/ops/math_ops_test.py

```diff
@@ -872,5 +872,29 @@ def testConvertToTensorRange(self):
     self.assertAllEqual(values, self.evaluate(tensor))
 
 
+@test_util.run_all_in_graph_and_eager_modes
```

Comment:
There already are tests for `math_ops.exp` in `python/kernel_tests/cwise_ops_unary_test.py`, for example:
https://github.com/tensorflow/tensorflow/blob/a366858377a2942b691c8664c10e142d8bca8452/tensorflow/python/kernel_tests/cwise_ops_unary_test.py#L206

Could you please add the additional test to `testBfloat16Basic` instead?
https://github.com/tensorflow/tensorflow/blob/a366858377a2942b691c8664c10e142d8bca8452/tensorflow/python/kernel_tests/cwise_ops_unary_test.py#L387

Reply:
Ok, updated